src/config/IAppConfig.ts

```diff
@@ -155,4 +155,5 @@ export interface IServerConfig {
     referenceGenomeVersion: string;
     skin_home_page_show_unauthorized_studies: boolean;
     skin_home_page_unauthorized_studies_global_message: string;
+    skin_mutation_table_namespace_column_show_by_default: boolean;
```

Comment:
maybe we make a string of default column IDs?  We can follow up with that but this could get silly.  I think this would actually be pretty useful.

Reply:
We can. Good idea actually. Lets do this in a subsequent PR.